bindings/python/iota_sdk/client/_high_level_api.py

```diff
@@ -1,14 +1,16 @@
 # Copyright 2023 IOTA Stiftung
 # SPDX-License-Identifier: Apache-2.0
 
-from iota_sdk.types.common import HexStr
+from typing import List, Optional
+from dataclasses import dataclass
+from iota_sdk.types.common import HexStr, json
 from iota_sdk.types.output import OutputWithMetadata
 from iota_sdk.types.output_id import OutputId
 from iota_sdk.types.common import CoinType
-from typing import List, Optional
-from dacite import from_dict
 
 
+@json
+@dataclass
 class Range:
     """Represents a range of address indexes.
```

Comment:
range still has an init method, we want those?

Reply:
removed, fixed